cloud/scope/cluster.go

```diff
@@ -54,7 +51,7 @@ func NewClusterScope(ctx context.Context, params ClusterScopeParams) (*ClusterSc
 	}
 
 	if params.GCPServices.Compute == nil {
-		computeSvc, err := createComputeService(ctx, params)
+		computeSvc, err := createComputeService(ctx, params.GCPCluster.Spec.CredentialsRef, params.Client)
```

Comment:
Caught my eye while exploring CAPG... IMO, we can add `ctx` init like `ctx := context.Background()` before passing it. 😄